src/lib.rs

```diff
@@ -296,6 +294,104 @@ pub fn resourceprep(s: &str) -> Result<Cow<'_, str>, Error> {
     Ok(Cow::Owned(normalized))
 }
 
+fn is_combining_char(c: char) -> bool {
+    matches!(c, '\u{0300}'..='\u{0345}' |
```

Comment:
Are there more reliable alternatives for this? This is just a snippet I've found while searching source codes for similar functionality (from github.com/webern/exile), and I've only added it here to check the tests